test/quantization/test_quantize_script.py

```diff
@@ -2574,3 +2600,231 @@ def forward(self, x):
         FileCheck().check_count("quantized::linear_dynamic(", 2, exactly=True) \
                    .check_not("aten::_choose_qparams_per_tensor") \
                    .run(m1.graph)
+
+class TestQuantizeScript(QuantizationTestCase):
+    def test_single_linear(self):
+        r"""Compare the result of quantizing single linear layer in
+        eager mode and graph mode
+        """
+        for qengine in supported_qengines:
+            with override_quantized_engine(qengine):
```

Comment:
can you use this decorator: https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_quantized.py#L119

you can search in the code base to see how this is used.